### ISSUE TITLE
Creative types

### ISSUE THREAD

**Issue by michaelbeil** [AffiliateWP/AffiliateWP#1087]
Per a convo with @sumobi it might be a great idea to have “creative types." You pick Facebook, then you enter the text, the link, and that's it. On the creatives section it shows a “Facebook” category with the text and link, and an image to download perhaps.

**Comment by sumobi** [comment]
Yeah, something worth exploring. The main reason for this is that an affiliate can't simply copy and paste the HTML provided from the creatives area, directly into Facebook. Facebook requires you to enter text, and a link separately. You also can't add an image, so perhaps we could provide guidelines on this new creative type and link to our doc which outlines OG tags etc.

**Comment by michaelbeil** [comment]
Related #448

**Comment by pippinsplugins** [comment]
Carrying type suggestions over from #1233 

- Emails
- Text
- Image
- HTML
- Twitter
- Facebook
- Downloadable documents, eg .docx or similar
- Etc

**Comment by pippinsplugins** [comment]
Some potential work done in https://github.com/AffiliateWP/AffiliateWP/pull/993

**Comment by gspice** [comment]
User ticket casting his vote to have videos as option to add as a Creative

https://secure.helpscout.net/conversation/305625276/51038/?folderId=205529

**Comment by gspice** [comment]
user ticket that would like to insert a javascript snippet that creates a form as a creative for his affiliates:

https://secure.helpscout.net/conversation/316978925/52652/?folderId=205529

**Comment by gspice** [comment]
Another for additional types: 

"Add more types of creatives like html banner, email html, text links, cover html"

https://secure.helpscout.net/conversation/325276861/53979/?folderId=205524

**Comment by amdrew** [comment]
From the issue above:

https://secure.helpscout.net/conversation/611937931/85148?folderId=634609.

**Comment by michaelbeil** [comment]
The same customer requested for HTML 5 banners again: https://secure.helpscout.net/conversation/615783153/85493?folderId=205524.

**Comment by michaelbeil** [comment]
The same customer requested HTML 5 banners again: https://secure.helpscout.net/conversation/621651590/86053/.
